textacy/text_stats.py

```diff
@@ -85,3 +85,20 @@ def coleman_liau_index(n_chars, n_words, n_sents):
 def automated_readability_index(n_chars, n_words, n_sents):
     """https://en.wikipedia.org/wiki/Automated_readability_index"""
     return 4.71 * (n_chars / n_words) + 0.5 * (n_words / n_sents) - 21.43
+
+
+def wiener_sachtextformel(n_words, n_sents, n_polysyllable_words, n_singlesyllable_words, n_long_words, variant=1):
```

Comment:
Would you mind adding parentheses around all of the subexpressions to make the order of operations more obvious? I don't see any errors, but I always have to remind myself how expressions like `X / Y * Z` actually evaluate.